packages/clay-card/stories/index.tsx

```diff
@@ -440,4 +458,159 @@ storiesOf('Components|ClayCard', module)
 				</ClayCard>
 			</ClayCard.Group>
 		</>
+	))
+	.add('ProductCard', () => (
+		<div className="row">
+			<div className="col-md-4">
+				<ClayCard className="product-card" displayType="file">
+					<ClayCard.AspectRatio className="card-item-first">
+						<div className="aspect-ratio-item aspect-ratio-item-center-middle aspect-ratio-item-fluid card-type-asset-icon">
+							<ClayIcon
+								spritemap={spritemap}
+								symbol="documents-and-media"
+							/>
+						</div>
+
+						<div className="aspect-ratio-item-bottom-left">
+							<ClayLabel displayType="success">
+								{'Available'}
+							</ClayLabel>
+						</div>
+					</ClayCard.AspectRatio>
+
+					<ClayCard.Body>
+						<div className="c-mb-3">
+							<div className="card-subtitle" title="AR385672">
+								<span className="text-truncate-inline">
+									<span className="text-truncate">
+										{'AR385672'}
+									</span>
+								</span>
+							</div>
+
+							<div
+								className="card-title"
+								title="thumbnail_coffee.jpg"
+							>
+								<span className="text-truncate-inline">
+									<a className="text-truncate" href="#1">
+										{
+											'ReallySuperInsanelyJustIncrediblyLongAndTotallyNotPossibleWordButWeAreReallyTryingToCoverAllOurBasesHereJustInCaseSomeoneIsNutsAsPerUsual'
+										}
+									</a>
+								</span>
+							</div>
+
+							<div className="card-text">
+								<s>{'$99.00'}</s>
+								<span>{'$70.00'}</span>
+							</div>
```

Comment:
I think you can use `<ClayCard.Description />` for this sections and using its different `displayType`'s available.

Reply:
Aah, yea I missed this because I was searching by `card-title`. I see now that it's implemented as `card-${displayType}`